tests/flow/test_udf.py

```
@@ -0,0 +1,1035 @@
+from common import *
```

Ruff (0.12.2)

`from common import *` used; unable to detect undefined names

(F403)